packages/x-components/src/x-modules/search/components/__tests__/promoteds-list.spec.ts

```diff
@@ -118,13 +118,13 @@ describe('testing PromotedsList component', () => {
   });
 
   it('provides the result of concatenating ancestor injected items with the promoteds', () => {
-    const resultStub = getResultsStub().slice(0, 1);
-    const promotedStub = getPromotedsStub().slice(0, 1);
+    const resultStub = getResultsStub();
+    const promotedStub = getPromotedsStub();
     const localVue = createLocalVue();
     localVue.use(Vuex);
     const store = new Store<DeepPartial<RootXStoreState>>({});
     installNewXPlugin({ store }, localVue);
-    resetXSearchStateWith(store, { promoteds: promotedStub });
+    resetXSearchStateWith(store, { promoteds: promotedStub, totalResults: resultStub.length * 2 });
```

Comment:
I see we have 4 results and 6 promoted, is it okay for the test to only load 9 items? didn't we talk about to concatenate all the promoted that are left behind if there are no more pages? or is it just true for the first one?

Reply:
For the case of the test the situation is that there are 4 results loaded but the total is 8 (4 * 2), so it is simulating the case that not all the results are already fetched. 

In that case, promoteds with a position out of the current page won't be displayed, and that's the case of the last promoted in the test (the one with position = 11).